### PR TITLE
Add CITATION.cff file

### LINKED ISSUE #460: add a CFF file
would replace the zenodo json file

### PR DESCRIPTION
**Pull request by cgobat**
Resolves #460.

Feel free to make updates as you see fit&mdash;I sourced this info from the existing [codemeta.json](../blob/main/joss/codemeta.json) and the JOSS publication itself.

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -0,0 +1,75 @@
+cff-version: 1.2.0
+title: 'SpiceyPy: a Pythonic Wrapper for the SPICE Toolkit'
+message: If you are publishing work that uses SpiceyPy, please cite SpiceyPy and the SPICE toolkit.
```
Comment (AndrewAnnex) [inline CITATION.cff:3]: should the message be in quotes? also update to say "cite SpiceyPy using the JOSS doi..." or something to the effect of if citing the use of spiceypy, use the joss doi, but if needing to cite a specific version of spiceypy, cite the specific zenodo doi.

File: CITATION.cff [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    orcid: 'https://orcid.org/0000-0002-0455-9384'
+  - given-names: Shankar
+    family-names: Kulumani
+    affiliation: Collins Aerospace
+    orcid: 'https://orcid.org/0000-0002-7822-0471'
+  - given-names: Shin-ya
+    family-names: Murakami
+    affiliation: GFD Dennou Club
+    orcid: 'https://orcid.org/0000-0002-7137-4849'
+identifiers:
```
Comment (AndrewAnnex) [inline CITATION.cff:66]: I wonder if it would be appropriate to add a references section to cite the NAIF here

Comment (AndrewAnnex) [reply CITATION.cff:66]: and I think maybe preferred-citation should be used with the JOSS doi

**Review by AndrewAnnex — COMMENTED** [review]
thanks for the contribution @cgobat, just a few small questions and a quick request to add https://github.com/marketplace/actions/cff-validator to the github workflows to validate the file where needed.

File: CITATION.cff (hunk 1 above)
Comment (cgobat) [reply CITATION.cff:3]: > should the message be in quotes?

No, YAML only needs quotes when the string contains disallowed characters, like `:`. I am actually removing a bunch of unnecessary quotes in the updates I will push shortly.

Comment (cgobat) [reply CITATION.cff]: Updated the message. Let me know if it's what you were going for.

File: CITATION.cff (hunk 2 above)
Comment (cgobat) [reply CITATION.cff:68]: Done & done.

**Comment by cgobat** [comment]
Anything else you wanted to see here, @AndrewAnnex?

**Pull request merged**